### PR TITLE
Solo mlorgain

### PR DESCRIPTION
**Pull request by dmilburn**

### CODE REVIEW

File: app/controllers/list.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+get '/lists' do
+	@all_lists = List.all
```
Comment (dmilburn) [inline app/controllers/list.rb]: `@lists`

File: app/controllers/list.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  	else
+    	@errors = list.errors.full_messages
+    	erb :'/lists/new'
+  	end
+end
+
+get '/lists/:id' do
+	if logged_in?
+		@list = List.find(params[:id])
+		@all_tasks = Task.where(list_id: @list.id)
```
Comment (dmilburn) [inline app/controllers/list.rb]: `@list.tasks`

File: app/views/users/show.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<div class="container">
+	<h1><a href='/'><%= @user.username %></a></h1>
+
+	<h2>Email: <%= @user.email %> </h2>
+	<h2>Total Lists Created: <%= @user.lists.count %> </h2>
+	<h2>Total Tasks Created: <%= @user.tasks.count %> </h2>
```
Comment (dmilburn) [inline app/views/users/show.erb:7]: `@lists = @user.lists`

**Comment by mlorgain** [comment]
@IuliiaKot @nyc-red-pandas-2016/mentors

**Pull request closed without merging**